### DISCUSSION TITLE
How to set proxy address ?

### DISCUSSION THREAD

**Discussion by patrice-conil** [quarkiverse/quarkus-vault#129] [Q&A] [answered]
Hi all,
I tried to set the proxy address using the jvm parameters (http.proxyHost and http.proxyPort) but it seems the vault client is not using them.
Am I doing something wrong?

Thanks in advance.
Patrice

**Comment by kdubb** [comment]
@gsmet Is there a standard way for Quarkus and its extensions to handle proxy configuration?

**Comment by patrice-conil** [comment, marked as answer]
Hi @gsmet, @kdubb  ,
I've proposed a PR #130 to handle this similar way as oidc. This PR has been merged. 
BR,
Patrice

**Comment by kdubb** [comment]
@patrice-conil Yes and thank you. I am wondering if there are global configuration values we should be using; similar to `quarkus.tls.trust-all`.

**Comment by patrice-conil** [comment]
Hi @kdubb,
Using global config could be problematic in my case (openshift deployment using internal and external services).
Having a way to set it for all traffic might be nice in a way, but we need to keep the specific way in parallel.
BR,
Patrice
